vars/mavenNode.groovy

```diff
@@ -5,6 +5,18 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    def nodesDir = "/var/lib/jenkins/nodes"
+    def file = new File(nodesDir)
+    
+    if(file.isDirectory()){
+        echo "$nodesDir exists and it's a directory"
+    } else {
+        error """
+        Directory $nodesDir does not exist. /nodes directory stores data for managing slave nodes.
```

Comment:
could we rephrase it:

```
"""
$nodesDir does not exist, without this directory Jenkins could not finish the build.
"""
```
Let's keep it simple rather than verbose technicality.

WDT?

Reply:
You are not rephrasing it. You are putting a message that has a very different meaning. 
Me and @sthaha had agreed on this message. Why are we trying to re-innovate when this PR is copy of an earlier [PR](https://github.com/fabric8io/fabric8-pipeline-library/pull/436/files) (that was approved and merged)?

Reply:
@kishansagathiya we can go ahead. Don't you feel this message is too verbose for a user?

If no, then let's go ahead and merge it.